### PR TITLE
W1 - D1 - trivo - git flow

### PR DESCRIPTION
**Pull request by at-trivo**
Task: #3
Please review me 
Thank you

### CODE REVIEW

File: app/src/at-trivo/res/values/strings.xml [hunk 1]
```diff
@@ -1,3 +1,5 @@
 <resources>
     <string name="app_name">Intern18Summer</string>
+    <string name="aasd"></string>
+    <string name="anotherString"></string>
```
Comment (at-chauhoang) [inline app/src/at-trivo/res/values/strings.xml]: Please remove string at line 3,4.

File: build.gradle [hunk 2]
```diff
@@ -7,7 +7,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.2'
+        classpath 'com.android.tools.build:gradle:3.1.3'
```
Comment (at-chauhoang) [inline build.gradle]: Please not change it.

File: app/src/at-trivo/res/trivo.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+* chơi game
+* xem bóng đá
+* chơi sodoku
+
+## Mục tiêu nghề nghiệp
+
+Trở thành technical leader,1 chuyên gia
+
+
+> the problem is not the problem ,the problem is your attitute about the problem
```
Comment (at-chauhoang) [inline app/src/at-trivo/res/trivo.md]: Please move trivo.md file like
 app/src/at-trivo/trivo.md

**Comment by at-chauhoang** [comment]
Please change decription 
```Task: #1``` to ```Task: #3```

**Comment by at-trivo** [comment]
i fixed it
plz review for me

**Pull request merged**